Cypress/cypress/integration/Parallel_Tests/Create_Workflow/validateWorkflowScore.spec.js

```diff
@@ -0,0 +1,72 @@
+// <reference types="Cypress" />
+import * as user from "../../../fixtures/Users.json";
+
+describe("Testing the validation of the final verdict and resilience score", () => {
+	before("Loggin in and checking if agent exists", () => {
+		cy.requestLogin(user.AdminName, user.AdminPassword);
+		cy.waitForCluster("Self-Agent");
+		cy.visit("/workflows");
+	});
+
+	it("Checking final verdict and resilience score", () => {
+		cy.GraphqlWait("workflowListDetails", "listSchedules");

```

Comment:
Can we add this test as a generic function?
It should be able to take arguments such as workflows name, agent name & expected result (RR score & workflow final status -->Succeded/Failed/Terminated/Running).
In the function, we can test all details accordingly for given workflow.
Being generic, it can be used in other test suites.